.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
I wish we could leave this on. If you're using VSCode with prettier enabled it should "just work", I'd like to get to the bottom of this. As discussed we can likely move to the tooling I've set up for Third Room.

Reply:
Oops - I didn't mean to commit this. My vote would probably be to not commit this though, as these are often down to people's individual preference. If we move to prettier, this should make vscode's format-on-save redundant anyway.

Reply:
Yup sounds good. I see the other linting PR which should fix this.